crates/extensions/c8y_config_manager/src/plugin_config.rs

```diff
@@ -193,7 +193,9 @@ impl PluginConfig {
 }
 
 #[derive(thiserror::Error, Debug)]
-#[error("The requested config_type {config_type} is not defined in the plugin configuration file.")]
+#[error(
+    "The requested config_type {config_type:?} is not defined in the plugin configuration file."
```

Comment:
It's nice to delimit values to make the value clear, and debug printing is the safest, and easiest, way to do this (for `String`s). It's a value that's coming from a user, so it's generally a good idea IMO to avoid any log injection risks (even if those are low).